dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/ValidationReport.java

```diff
@@ -59,16 +59,20 @@ public static ValidationReport emptyReport()
         return new ValidationReport();
     }
 
-    public static ValidationReport merge( ValidationResult result, ValidationResult anotherResult )
+    public static ValidationReport merge( ValidationResult validationResult, ValidationResult anotherValidationResult )
     {
-        return new ValidationReport();
+        ValidationReport validationReport = fromResult( validationResult );
```

Comment:
Can we improve the API so that we just do `ValidationReport validationReport = fromResult( validationResult, anotherValidationResult);`? So there is no mutation of the ValidationReport